### DISCUSSION TITLE
Place component beside tooltip

### DISCUSSION THREAD

**Discussion by kaisiant** [reactivemarkets/react-financial-charts#480] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/26655111/107313082-ca030400-6acc-11eb-88f6-3e20216b74c3.png)
Hi, I want to add the buttons/text beside the tooltip, like the picture beow. Is there any way to do it?
![image](https://user-images.githubusercontent.com/26655111/107313114-dab37a00-6acc-11eb-9fd1-6ff1ff0a24d8.png)

**Comment by markmcdowell** [comment, marked as answer]
It's not supported by the current tooltips, you'd have to create your own/raise a PR for it.

Thanks.
